site/src/components/NavbarView/NavbarView.tsx

```diff
@@ -124,13 +173,25 @@ const useStyles = makeStyles((theme) => ({
 
       "&::before": {
         content: `" "`,
-        bottom: 0,
-        left: theme.spacing(3),
+        left: 0,
+        width: 2,
+        height: "100%",
         background: theme.palette.secondary.dark,
-        right: theme.spacing(3),
-        height: 2,
         position: "absolute",
+
+        [theme.breakpoints.up("md")]: {
+          bottom: 0,
+          left: theme.spacing(3),
+          width: `calc(100% - 2 * ${theme.spacing(3)}px)`,
```

Comment:
`nit` A comment describing the calculation here would be nice.

Reply:
I put the commit on auto merge but I will address this when I touch it again.